database/database.sql

```diff
@@ -0,0 +1,94 @@
+/*To create this database locally in your system  follow the following steps
+
+	1. Install PostgreSQL locally in your system . Follow the following steps to intsall it if not installed in ubuntu
+		1.Open the terminal (Ctrl+Alt+T)
+		2. Type $ sudo apt-get update
+		3. Type $ sudo apt-get install postgresql postgresql-contrib 
+	2. Switch over postgres account on your server by typing 
+		$ sudo -i -u postgres
+	3. Now access the postgres prompt by typing 
+		$ psql
+
+	4.Now create the database 'Transport_Mangement_System' by typing the following command
+		CREATE DATABASE Transport_Management_System ;
+
+	5. Type the following command to connect/select this database
+		# \c Transport_Mangement_System
+	
+	6. To Create the relations locallcy in the system run the following queries in your system*/ 
+
+CREATE TABLE Stu_Per_Data (
+	USN VARCHAR PRIMARY KEY NOT NULL,
+	Name CHAR(30) NOT NULL,
+	Email VARCHAR NOT NULL,
+	Mobile_No VARCHAR NOT NULL,
+	Status boolean NOT NULL DEFAULT false,
+	latitude real,
+	longitude real,
+	password VARCHAR 
+);
+
+CREATE TABLE USN_UID(
+	USN VARCHAR REFERENCES Stu_Per_Data(USN),
+	UID Serial NOT NULL PRIMARY KEY 
+);
+
+
+CREATE TABLE Driver (
+	Driver_id INT PRIMARY KEy,
+	Driver_name CHAR(50),
+	Mobile_No VARCHAR ,
+	password VARCHAR
+);
+
+CREATE TABLE Bus (
+	Bus_no VARCHAR PRIMARY KEY ,
+	Capacity INT 
+);
+
+CREATE TABLE Trip (
```

Comment:
Here, a single route and one bus can have many trips at different time right? 
If yes, then route_no, bus_no, timings,date should become a single primary key.
And we should add date also right?If we store one week trip details?